src/response.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * Utility module that converts squelize models into
+ * Sensorthings format.
+ */
+
+const FIELDS_TO_FILTER = [
+  'id',
+  'createdAt',
+  'updatedAt'
+];
+
+function generate(resource) {
+  let response = {};
+  if (Array.isArray(resource)) {
+    response['@iot.count'] = resource.length;
+    response.value = [];
+    resource.forEach((item) => {
+      response.value.push(_formatItem(item));
+    });
+    return response;
+  }
+
+  return _formatItem(resource);
+}
+
+function _formatItem(item) {
+  const itemData = item.dataValues;
+  const resourceName = item['$modelOptions'].name.plural;
+  let formatedItem = {
+    '@iot.id': itemData.id,
+    '@iot.selfLink': '/' + resourceName +'/' + itemData.id
+  };
+
+  Object.keys(itemData).forEach((key) => {
+    if (FIELDS_TO_FILTER.indexOf(key) === -1) {
```

Comment:
I think it's more clear to inline the array here.

`if (['id', 'createdAt', 'updatedAt'].indexOf(key) === -1) {`